### DISCUSSION TITLE
Two Linux servers on one maschine

### DISCUSSION THREAD

**Discussion by Buggy1987** [PR3SIDENT/enshrouded-server#48] [Q&A]
Hello
I used the commands from the script InstallServer.sh. It works fine. Now my question is: can I start two servers ?

**Comment by Buggy1987** [comment]
Sorry for asking again: Is there a solution to run 2 or more servers?

**Reply by WildPenquin** [reply]
Yes. Copy the installation to another folder to another folder (use hard links to save space). Delete the json from the other copy (to make sure the hard link is broken) and re-create, change saveDirectory and ports, and you are good to go (haven't tested this)!

I can not find a way to give another configuration file to the exe, and it does not seem to respect the working directory but look for the json file in the same directory as where the exe is. So that's why this kind of shenanigans is needed. You could ask for a feature (give configuration file as a parameter) from Keen games.

If you are using a docker image, it's (a tiny bit) simpler; just use different ports and two docker instances. Of course there will be overhead (of disk space).

Do note that the server requires a lot of CPU, unless you have a >12 core CPU (or a multi-CPU rack server or something) running many servers might not be a sensible thing to do.
